### DISCUSSION TITLE
Change default attach rule to current update rule

### DISCUSSION THREAD

**Discussion by stepanenko3** [laravel/nova-issues#5493] [Ideas & Feature Requests] [answered]
class: `Laravel\Nova\Authorizable`
methods: `authorizedToAttachAny`, `authorizedToAttach`, `authorizedToDetach`

I suggest changing the `true`, which is returned if there is no method in the Policy to: `$this->authorizedToUpdate($request)`

**Comment by crynobone** [comment, marked as answer]
This is a huge breaking change.
